src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/RecoveryPointConversions.cs

```diff
@@ -61,7 +64,7 @@ public class RecoveryPointConversions
                         RecoveryPointId = rp.Name,
```

Comment:
See if we can put common part of conversion between List and Get into another function